.ci/azure/linux_arm64.yml

```diff
@@ -138,6 +138,7 @@ jobs:
         -DENABLE_DATA=OFF
         -DCMAKE_TOOLCHAIN_FILE=$(OPENVINO_REPO_DIR)/cmake/arm64.toolchain.cmake
         -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DTHREADING=TBB
```

Comment:
why do we need to set it explicitly? 
I supposed it should be a default

Reply:
This change is just for testing and will be removed, there are some compilations which set THREADING=SEQ and I want to test compiling with TBB.